src/main/scala/StructuralTypes.scala

```diff
@@ -0,0 +1,36 @@
+
+/**
+  * Structural Types: http://dotty.epfl.ch/docs/reference/changed/structural-types.html
+  */
+object StructuralTypes {
+
+  /*
+    trait Selectable extends Any {
+      def selectDynamic(name: String): Any
+      def selectDynamicMethod(name: String, paramClasses: ClassTag[_]*): Any =
+        new UnsupportedOperationException("selectDynamicMethod")
+    }
+   */
```

Comment:
nope, the link actually contains that but I thought it would be fine to put it as documentation to know what is that Selectable. I can remove it if you don't like.